draft-ietf-ppm-dap.md

```diff
@@ -1947,6 +1947,11 @@ struct {
 computed above and `encrypted_aggregate_share.ciphertext` is the ciphertext
 `encrypted_agg_share` computed above.
 
+The helper's handling of this request MUST be idempotent. That is, if multiple
```

Comment:
I think a MUST (rather than a SHOULD) is appropriate here. If this is optional, then there's no way for the leader to recover from losing the first response to `POST /tasks/{task-id}/aggregate_shares`, because the helper can refuse to service subsequent aggregate share requests due to exhaustion of the max batch query count.

Reply:
I think this is roughly equivalent to saying that `max_batch_query_count` consumption is based on _distinct_ `AggregateShareReq`s, which is what we want, so 👍🏻.